experimental/dds/attributable-map/api-report/attributable-map.api.md

```diff
@@ -25,7 +25,6 @@ export class AttributableMap extends SharedObject<ISharedMapEvents> implements I
     [Symbol.iterator](): IterableIterator<[string, any]>;
     readonly [Symbol.toStringTag]: string;
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
-    // (undocumented)
```

Comment:
Anything that was previously documented using `{@inheritDoc }` pointing to a target in another library would be incorrectly annotated as `(undocumented)`. This is now fixed!